### PR TITLE
feat: add gap, gapColumn & gapRow styles

### PR DESCRIPTION
**Pull request by JadlionHD**
#74

### CODE REVIEW

**Review by DeVoresyah — CHANGES_REQUESTED** [review]
please also provide new custom theme key for gap

**Comment by DeVoresyah** [comment]
## Example
### Custom Theme
```
{
  "gap": {
     97: 388
   }
}
```

### Output Avaibility
- gap-97
- gap-x-97
- gap-y-97

**Comment by JadlionHD** [comment]
Thanks for the feedback, here's testing development react native web for custom theme gaps
image

File: src/processor/gaps.ts
Comment (DeVoresyah) [inline src/processor/gaps.ts:1]: please use `.reduce` to make it more simple and clean. You can refer to `letterSpacings.ts`

**Review by DeVoresyah — CHANGES_REQUESTED** [review]
minor changes needed

File: src/processor/gaps.ts
Comment (JadlionHD) [reply src/processor/gaps.ts:1]: Sure, done

**Review by DeVoresyah — APPROVED** [review]
LGTM~!

**Pull request merged**